core/domain/user_services.py

```diff
@@ -172,8 +310,14 @@ def get_user_id_from_username(username):
 def get_user_settings_from_username(username):
     """Gets the user settings for a given username.
 
-    Returns None if the user is not found.
+    Args:
+        username: str. Identifiable username to display in the UI.
+
+    Returns:
+        The UserSettingsModel instance corresponding to the given username.
```

Comment:
Start with type information:

UserSettingsModel or None. The UserSettingsModel instance corresponding to the given username, or None if no such model was found.